src/Elasticsearch.Net/Configuration/Security/BasicAuthorizationCredentials.cs

```diff
@@ -1,13 +1,17 @@
-﻿namespace Elasticsearch.Net
+﻿using System;
+
+namespace Elasticsearch.Net
 {
 	public class BasicAuthenticationCredentials
 	{
-		public string UserName { get; set; }
+		public string Username { get; set; }
+		[Obsolete("Scheduled to be removed use Username instead, note the lowercase n")]
```

Comment:
Ah, scratch that.  It wasn't a rename :).  I did notice that the file name is still `Authorization` though, which should be `Authentication`.